conftest.py

```diff
@@ -147,30 +192,55 @@ def setup_artifact_path(item: pytest.Item, artifact_name: str):
     return os.path.join(output_dir, artifact_name)
 
 
-def get_rotating_task_log_lines(task_id: str, known_task_files: set, task_file: str):
+def get_task_files_for_id(task_id: str):
+    try:
+        return set(subprocess.check_output(['dcos', 'task', 'ls', task_id, '--all']).decode().split())
+    except:
+        log.exception('Failed to get list of files for task: {}'.format(task_id))
+        return set()
+
+
+def get_task_log_for_id(task_id: str,  task_file: str='stdout', lines: int=1000000):
+    log.info('Fetching {} from {}'.format(task_file, task_id))
+    result = subprocess.run(
+        ['dcos', 'task', 'log', task_id, '--all', '--lines', str(lines), task_file],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+    )
+    if result.returncode:
+        errmessage = result.stderr.decode()
+        if not errmessage.startswith('No files exist. Exiting.'):
+            log.error('Failed to get {} task log for task_id={}: {}'.format(task_file, task_id, errmessage))
+        return None
```

Comment:
The return type here is inconsistent. Should we not return `""` instead?

Reply:
Done.